stdin-stdout/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "jsonrpc-stdin-stdout-server"
```

Comment:
maybe "jsonrpc-stdio-server" would be a better name?

Reply:
Good point, I renamed it.